hackathon_site/dashboard/frontend/src/testing/mockData.tsx

```diff
@@ -16,6 +16,75 @@ import { CartItem } from "api/types";
 import { adminGroup } from "constants.js";
 
 // For DashCard on Dashboard
+
+export const mockOverviewStats = [
+    {
+        title: "checked_out",
+        quantity: 123,
+    },
+    {
+        title: "participants",
+        quantity: 200,
+    },
+    {
+        title: "teams",
+        quantity: 14,
+    },
+    {
+        title: "orders",
+        quantity: 123,
+    },
+    {
+        title: "broken",
+        quantity: 7,
+    },
+];
+
+export const mockAdminPendingOrders = [
```

Comment:
This array should be of type `Order[]` take a look at mockPendingOrders constant in this file, this should be the same structure as that, you can even reuse other order constants in this file instead of making a new one